tests/test-cases/Group2-Docker-Archive/2-01-Docker-Archive.robot

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 2-01 - Docker Archive
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  debug=3
+Suite Teardown  Cleanup VIC Appliance Together With tmp dir
+
+*** Keywords ***
+Cleanup VIC Appliance Together With tmp dir 
+    Cleanup VIC Appliance On Test Server
+    Run  rm -rf /tmp/compare
+    Run  rm -rf /tmp/pull
+    Run  rm -rf /tmp/save
+
+Compare Tar File Content
+    [Arguments]  ${image}  ${tarA}  ${tarB}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  tar -tvf ${tarA} > /tmp/compare/${image}/a
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Empty  ${output}
+    ${out}=  Run  cat /tmp/compare/${image}/a
+    Log  ${out}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  tar -tvf ${tarB} > /tmp/compare/${image}/b
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Empty  ${output}
+    ${out}=  Run  cat /tmp/compare/${image}/b
+    Log  ${out}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  diff /tmp/compare/${image}/a /tmp/compare/${image}/b
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Be Empty  ${output}
```

Comment:
It's here to do more work for the diff output.
We still could not get actually file content consistent, because we generated different .wh... file with docker.